runtime/lua/vim/treesitter/languagetree.lua

```diff
@@ -38,6 +38,8 @@ local LanguageTree = {}
 
 LanguageTree.__index = LanguageTree
 
+--- @private
+---
 --- A |LanguageTree| holds the treesitter parser for a given language {lang} used
```

Comment:
It seems like `LanguageTree.new` and `TSHighlighter.new` were being used as "module-level docstrings". Do we not have something like `\defgroup` for Lua modules? https://github.com/neovim/neovim/blob/6d4f48182131c36d57589eefd4cefe3c70256d04/src/nvim/api/buffer.c#L47

Reply:
Revisiting this in https://github.com/neovim/neovim/pull/22498